src/macaron/dependency_analyzer/cyclonedx_mvn.py

```diff
@@ -64,11 +64,20 @@ def collect_dependencies(self, dir_path: str) -> dict[str, DependencyInfo]:
         """
         # Load the top level file separately as it has different content.
         top_path = Path(os.path.join(dir_path, "target", self.file_name))
+        top_path_altered = False
+        if not os.path.exists(top_path):
+            # If the expected bom file does not exist, allow other named .json files instead.
+            possible_paths = glob.glob(os.path.join(dir_path, "target", "*.json"))
+            if possible_paths:
+                top_path = Path(possible_paths[0])
```

Comment:
If we are not going to support the case where multiple BOM files are generated into the `target` directory, should we exit on error here if `possible_paths` have multiple files, instead of picking the first element in `possible_paths`, which could not be the top BOM file we expect?